tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/Constants.java

```diff
@@ -35,6 +31,7 @@
     public static final String LISTENER_HANDLER = "listenerHandler";
     public static final String READ_TIMEOUT_HANDLER = "readTimeoutHandler";
     public static final String CLIENT_HANDLER = "clientHandler";
+    public static final String SSL_HANDLER = "SSL_Handler";
 
     // remote method names
```

Comment:
```suggestion
    // Remote method names
```
Always start such phrases with capital letters.

Reply:
updated